sidecar/config/config.go

```diff
@@ -124,15 +124,15 @@ type HealthCheck struct {
 	Method     string        `yaml:"method"`
 	Code       int           `yaml:"code"`
 	Args       []string      `yaml:"args"`
-	CACertPath string        `yaml:"ca_cert_path"`
+	CACertFile string        `yaml:"ca_cert_file"`
 }
 
 // ProxyConfig stores proxy configuration.
 type ProxyConfig struct {
 	TLS              bool   `yaml:"tls"`
-	CertPath         string `yaml:"cert_path"`
-	CertKeyPath      string `yaml:"cert_key_path"`
-	CACertPath       string `yaml:"ca_cert_path"`
+	CertChainFile    string `yaml:"cert_chain_file"`
+	PrivateKeyFile   string `yaml:"private_key_file"`
+	CACertFile       string `yaml:"ca_cert_file"`
```

Comment:
One thought: do we want to move TLS into its own struct? Then we'd just check whether there was a TLS struct instead of checking the TLS bool.